lib/fathead/bower/output.txt

```diff
@@ -0,0 +1,24 @@
+cache	A										<section class="prog__container">Manage bower cache<pre><code>\n$ bower cache <command> [<args>]\n</code></pre></section>	https://bower.io/docs/api/#cache
```

Comment:
@NamPNQ unfortunately these article titles are simply too generic.

If we were to use this Fathead as-is, we'd be saying that when someone searches "cache", "help", "home", etc. we'd be showing a portion of Bower documentation -- which you can imagine would be quite irrelevant to most people.

Let's preface each of these article titles with the word "bower" so our article titles are "bower cache", "bower home", "bower info", etc.

Even still I'm worried some of these may be too generic. Someone searching "bower help" probably isn't looking to know the syntax of the `bower help` command, but rather looking for help resources related to bower.